### PR TITLE
textlint-rule-prh: init at 6.0.0

### PR DESCRIPTION
**Pull request by natsukium**
## Description of changes

[textlint](https://github.com/textlint-rule/textlint) rule for [prh/prh: proofreading helper](https://github.com/prh/prh).

This rule check the spell by used with prh.yml.

https://github.com/textlint-rule/textlint-rule-prh

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/te/textlint-rule-prh/package.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  src = fetchFromGitHub {
+    owner = "textlint-rule";
+    repo = "textlint-rule-prh";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-K2WkHh7sLnhObM2ThvdXVbZymLInjSB6XTshxALotKU=";
+  };
+
+  postPatch = ''
+    substituteInPlace package.json \
+      --replace "git config --local core.hooksPath .githooks" ""
```
Comment (kirillrdy) [inline pkgs/by-name/te/textlint-rule-prh/package.nix:28]:
```suggestion
      --replace-fail "git config --local core.hooksPath .githooks" ""
```

File: pkgs/by-name/te/textlint-rule-prh/test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+オープンソースソフトウェア
```
Comment (kirillrdy) [inline pkgs/by-name/te/textlint-rule-prh/test.md:1]: no need to change, but you can inline this

File: pkgs/by-name/te/textlint-rule-prh/package.nix (hunk 1 above)
Comment (natsukium) [reply pkgs/by-name/te/textlint-rule-prh/package.nix:28]: good catch, thanks.

File: pkgs/by-name/te/textlint-rule-prh/test.md (hunk 2 above)
Comment (natsukium) [reply pkgs/by-name/te/textlint-rule-prh/test.md:1]: A separate file is also helpful for quick debugging.

**Comment by kirillrdy** [comment]
Result of `nixpkgs-review pr 330122` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>textlint-rule-prh</li>
  </ul>
</details>

not sure why ofborg is having issues

**Comment by natsukium** [comment]
`textlint` is broken on aarch64-linux, and a test that depends on it seems to have failed.

**Pull request merged**